env.development

```diff
@@ -0,0 +1,4 @@
+GATSBY_WEBSITE_URL=http://localhost:8100
```

Comment:
this shouldn't be here, should be a dot file too, right? ;) That way it's caught by `.gitignore`

Reply:
removed.